### DISCUSSION TITLE
How to configure PAT token using credentials in Linux server

### DISCUSSION THREAD

**Discussion by MeerajohnA** [community/community#102848] [Programming Help]
### Select Topic Area

Bug

### Body

I have configured pat token with git credentials in linux but I am unable to clone

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MeerajohnA ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
